paasta_tools/paasta_remote_run.py

```diff
@@ -101,6 +129,22 @@ def main(argv):
     if command:
         overrides_dict['cmd'] = command
 
+    run_id = args.run_id
+    if run_id is None:
+        run_id = ''.join(
+            random.choice(string.ascii_uppercase + string.digits) for _ in range(8))
+        paasta_print("Assigned random run-id: %s" % run_id)
+
+    if args.detach:
```

Comment:
sort of for completeness. ctrl-z would still make remote-run rely on ssh connection flakiness. also considering how little useful output remote-run script generates in stdout (basically "TASK_RUNNING, TASK_FINISHED") i think detached runs will be the norm.